src/views/EditorView/EditorContainer/EditorContainer.tsx

```diff
@@ -168,9 +168,9 @@ const EditorContainer: React.FC<IProps> = ({
                         })
                         .filter((style) => style !== 0)
                         .map((style) => {
-                            // console.log('style == ', style);
+                        // console.log('style == ', style);
```

Comment:
It would be nice if you kept the previous code when it is not related to this PR.